### DISCUSSION TITLE
HOW TO SETUP RUSTDESK SERVER ON QNAP USING CONTAINER STATION

### DISCUSSION THREAD

**Discussion by chrisphy00** [rustdesk/rustdesk#5774] [General]
Is there any person? who have expreience to setup the rustdesk server on qnap NAS using container station. Can anyone share there setup expreience? First thanks for all.

**Comment by eli0te** [comment]
Hi there,

I am currently trying without success for the moment

I just created a new application in container station using [this docker compose](https://rustdesk.com/docs/en/self-host/rustdesk-server-oss/docker/#docker-compose-examples)
Finally open my ports 21115-21117 (TCP) and 21116 UDP in my firewall and port forwarding though my router.

Now trying some stuff to connect but it appear not to work remotely for now.

**Comment by AriaRoba** [comment]
Follow instructions in this video. 
https://www.youtube.com/watch?v=PjY5TpWMNz4
It works.

**Reply by jey328** [reply]
Watched but failed to make it work, can’t understand the language 😬
